lib/rubocop/cop/sorbet/signatures/signature_build_order.rb

```diff
@@ -12,7 +9,11 @@
 module RuboCop
   module Cop
     module Sorbet
-      class SignatureBuildOrder < SignatureCop
+      #
+      #
+      class SignatureBuildOrder < ::RuboCop::Cop::Cop # rubocop:todo InternalAffairs/InheritDeprecatedCopClass
```

Comment:
This cop is currently undocumented. Without the empty comments, YARD uses the `rubocop:disable` comment as documentation, which makes it into `manual/cops_sorbet.md`. Adding these explicitly provides empty documentation, which maintains the status quo, as documenting it is out of scope of this PR.